scipy/optimize/_minimize.py

```diff
@@ -19,7 +19,8 @@
                         _minimize_bfgs, _minimize_newtoncg,
                         _minimize_scalar_brent, _minimize_scalar_bounded,
                         _minimize_scalar_golden, MemoizeJac, OptimizeResult,
-                        _wrap_callback)
+                        _wrap_callback, BracketError,
+                        _recover_from_bracket_error)
```

Comment:
```suggestion
                        _wrap_callback, _recover_from_bracket_error)
```
We can commit this with `[skip ci]` when this is ready to be merged.